relayer-test/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name       = "ibc-relayer-test"
```

Comment:
I now call it `ibc-integration-test`. Right now I setup the `tracing_subscriber` in the test so that it only show logs from crates with the `ibc_` prefix, so the test crate itself also have to start with `ibc-`.